templates/downloads/release_detail.html

```diff
@@ -49,6 +56,9 @@ <h1 class="page-title">Files</h1>
               <th>MD5 Sum</th>
               <th>File Size</th>
               <th>GPG</th>
+              {% if has_sigstore_materials %}
+              <th colspan="2">Sigstore</th>
```

Comment:
Would it be a bad idea to link from this to some reference on what sigstore is?

Reply:
That's the plan! I'll get that page up before we merge this, and add the link here.